test/init.js

```diff
@@ -12,11 +12,11 @@ let DataSource = juggler.DataSource;
 const config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
 global.getConfig = function(options) {
   const dbConf = {
-    host: process.env.MYSQL_HOST || config.host || 'localhost',
+    host: process.env.MYSQL_HOST || config.host || '127.0.0.1',
```

Comment:
I think forcing the IPv4 localhost is a good idea and we can land this change independently on the rest :+1: